src/components/fx/texttemplate_attributes.js

```diff
@@ -0,0 +1,53 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var FORMAT_LINK = require('../../constants/docs').FORMAT_LINK;
+
+module.exports = function(opts, extra) {
```

Comment:
`texttemplate` isn't really an _fx_, could you move this file to `plots/texttemplate_attributes.js`?

Reply:
Done in ce3c9cccca5b9ae3a2408681463a3b2f49d903ca